config.json

```diff
@@ -331,7 +331,13 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "conditionals",
+        "loops",
+        "integers",
+        "strings",
+        "maps",
```

Comment:
Hm, this is interesting. `<class 'dict'>` in Python is an essence a `Map`, but newcomers might not even realize it.
Thoughts?

I guess we can leave it

Reply:
That was my original thinking, that newcomers would be thinking "map" from other languages, so it might help with the transition. Thinking about it more, perhaps best to remove it to avoid confusion with the `Map` function.